### PR TITLE
Laetitia Rework

### PR DESCRIPTION
**Pull request by PotatoTomahto**
## About The Pull Request

Changes how Laetitia's prank works.
- 70% to gain the gift on Neutral is unchanged
- 30% to remove gift on Good work, 15% to remove gift and 15% to begin explosion on Neutral, 70% to begin explosion on Bad
- If you work another abnormality with the gift, the explosion timer is halved
- Randomed gift timer between 3 to 4 minutes
- If you have Prudence 60, you get at most a 10 second warning before explosion.
- If you have Prudence 80, you are notified when you get the gift.

Added gift message and a neat indicator as a warning

Adds info to records, fixes consistency of some HE records.
## Why It's Good For The Game
Her prank used to be a 200 damage that was given out at a high chance, forcing people to stop their entire grind and wait it out with red armor. Because this was way too unpredictable and punishing, she was rarely chosen and worked significantly less than other HEs. This PR is designed to give people a chance to deal with and react to the prank, while still keeping the RNG surprise element. 
Laetitia is now an abnorm that punishes you if you're given her gift and work other abnorms, while also encouraging players to continue working her. Basically, be ready to keep working with her if you get a neutral ~~because she's a good girl who does nothing wrong~~.

## Changelog
:cl:
tweak: laetitia
:cl:

### CODE REVIEW

**Comment by Kitsunemitsu** [comment]
Remove the warning time, it should be a jumpscare.

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
See above comment.

**Comment by Kitsunemitsu** [comment]
Waiting on the rework from REDACTED, honestly. IDK Why you didn't wait until his PR was merged like I recommended.

File: code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm:37]: That signal should probably go to the status effect itself.

File: code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm:103]: Shouldn't there be parent call?

File: code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm:121]: Same here. Missing parent call, I assume.

File: code/modules/paperwork/records/info/waw.dm
Comment (EgorDinamit) [inline code/modules/paperwork/records/info/waw.dm:325]: You accidentally deleted records...

File: code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm
Comment (dsmith328) [inline code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm:125]: `<=` might be better here

File: code/modules/paperwork/records/info/he.dm
Comment (dsmith328) [inline code/modules/paperwork/records/info/he.dm:249]: please remove the word `enters`

File: code/modules/paperwork/records/info/waw.dm
Comment (dsmith328) [reply code/modules/paperwork/records/info/waw.dm:325]: See now it's gone... forever gone!

**Review by EgorDinamit — DISMISSED** [review]

**Review by EgorDinamit — APPROVED** [review]

**Review by Kitsunemitsu — APPROVED** [review]
Yeah if it's 10 seconds it's good.
I approve this from a balance perspective

**Pull request merged**
